server/sonar-server/src/main/java/org/sonar/server/qualityprofile/BuiltInQProfileInsertImpl.java

```diff
@@ -81,19 +81,19 @@ public void create(DbSession dbSession, DbSession batchDbSession, BuiltInQProfil
     Date now = new Date(system2.now());
     RulesProfileDto ruleProfile = insertRulesProfile(dbSession, builtInQProfile, now);
 
-    List<ActiveRuleChange> localChanges = builtInQProfile.getActiveRules()
+    List<ActiveRuleChange> changes = builtInQProfile.getActiveRules()
       .stream()
       .map(activeRule -> insertActiveRule(dbSession, ruleProfile, activeRule, now.getTime()))
       .collect(MoreCollectors.toList());
 
-    localChanges.forEach(change -> dbClient.qProfileChangeDao().insert(batchDbSession, change.toDto(null)));
+    changes.forEach(change -> dbClient.qProfileChangeDao().insert(batchDbSession, change.toDto(null)));
 
     associateToOrganizations(dbSession, batchDbSession, builtInQProfile, ruleProfile);
 
-    dbSession.commit();
+    // TODO batch statements should be executed through dbSession
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)